homeassistant/components/atome/manifest.json

```diff
@@ -3,6 +3,6 @@
   "name": "Atome Linky",
   "documentation": "https://www.home-assistant.io/integrations/atome",
   "codeowners": ["@baqs"],
```

Comment:
Don't forget to add you to code owners ;)

Reply:
Thank you very much :-)